.github/workflows/avm.template.module.yml

```
@@ -178,6 +178,7 @@ jobs:
           managementGroupId: "${{ secrets.ARM_MGMTGROUP_ID }}"
           removeDeployment: "${{ fromJson(inputs.workflowInput).removeDeployment }}"
           customLocation: "${{ fromJson(inputs.workflowInput).customLocation }}"
+          e2eignore: "${{ matrix.testCases.e2eignore }}"
```

```suggestion
          e2eIgnore: "${{ matrix.testCases.e2eIgnore }}"
```
given that all other parameters etc. are camel-cased...